[2023-03-29 15:42:48,612] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.start_task manual__2023-03-29T15:42:47.481903+00:00 [queued]>
[2023-03-29 15:42:48,623] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.start_task manual__2023-03-29T15:42:47.481903+00:00 [queued]>
[2023-03-29 15:42:48,623] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 15:42:48,623] {taskinstance.py:1357} INFO - Starting attempt 1 of 4
[2023-03-29 15:42:48,623] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 15:42:48,639] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): start_task> on 2023-03-29 15:42:47.481903+00:00
[2023-03-29 15:42:48,643] {standard_task_runner.py:52} INFO - Started process 134 to run task
[2023-03-29 15:42:48,645] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_merge_dag', 'start_task', 'manual__2023-03-29T15:42:47.481903+00:00', '--job-id', '526', '--raw', '--subdir', 'DAGS_FOLDER/data_merge.py', '--cfg-path', '/tmp/tmp3quksno3', '--error-file', '/tmp/tmp295hhemv']
[2023-03-29 15:42:48,645] {standard_task_runner.py:80} INFO - Job 526: Subtask start_task
[2023-03-29 15:42:48,700] {task_command.py:369} INFO - Running <TaskInstance: data_merge_dag.start_task manual__2023-03-29T15:42:47.481903+00:00 [running]> on host 5d8703454903
[2023-03-29 15:42:48,781] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kartaca
AIRFLOW_CTX_DAG_ID=data_merge_dag
AIRFLOW_CTX_TASK_ID=start_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T15:42:47.481903+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T15:42:47.481903+00:00
[2023-03-29 15:42:48,781] {logging_mixin.py:115} INFO - DAG has been started...
[2023-03-29 15:42:48,781] {python.py:173} INFO - Done. Returned value was: DAG has been started...
[2023-03-29 15:42:48,812] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_merge_dag, task_id=start_task, execution_date=20230329T154247, start_date=20230329T154248, end_date=20230329T154248
[2023-03-29 15:42:48,858] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-29 15:42:48,900] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
